### PR TITLE
refactor(title): add helmet and refactor titles

### PR DESCRIPTION
**Pull request by zouxuoz**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -40,6 +40,7 @@
     "react-addons-css-transition-group": "^15.3.2",
     "react-datepicker": "^0.30.0",
     "react-dom": "^15.3.1",
+    "react-helmet": "^3.1.0",
```
Comment (rosendi) [inline package.json]: Add it to the prod and dev configs to `vendor`

File: src/components/RootHeader/RootHeader.js [hunk 2]
```diff
@@ -1,6 +1,7 @@
 import React, { PropTypes } from 'react';
 import { FormattedMessage, defineMessages, intlShape } from 'react-intl';
 import { Link } from 'react-router';
+import Helment from 'react-helmet';
```
Comment (rosendi) [inline src/components/RootHeader/RootHeader.js]: Do you mean Helment

File: src/components/RootHeader/RootHeader.js [hunk 3]
```diff
@@ -75,8 +81,9 @@ const messages = defineMessages({
   },
 });
 
-const RootHeader = ({ block, element, intl, showNavigation, username }) => (
+const RootHeader = ({ block, element, intl: { locale, formatMessage }, showNavigation, username }) => (
   <header {...block()}>
+    <Helment titleTemplate={`${formatMessage(messages.title)} | %s`} />
```
Comment (rosendi) [inline src/components/RootHeader/RootHeader.js]: Add a new line

**Comment by rosendi** [comment]
lgtm

**Comment by radiansz** [comment]
lgtm

**Pull request merged**